src/ci_providers/provider_circleci.ts

```diff
@@ -30,22 +30,19 @@ function _getSHA(inputs: UploaderInputs): string {
 
 function _getSlug(inputs: UploaderInputs): string {
   const { args, environment: envs } = inputs
-  let slug = args.slug || ''
-  if (slug !== '') {
-    return slug
+  if (args.slug !== '') {
+    return args.slug
   }
```

Comment:
```suggestion
  const slug = setSlug(args.slug, envs.CIRCLE_PROJECT_USERNAME, envs.CIRCLE_PROJECT_REPONAME)
```

Reply:
I left this one alone because it has an extra check directly below it. The tests got super cranky when I touched it.